test/unit/spec/v2/ion/responseTransformer_spec.js

```diff
@@ -1,237 +1,425 @@
 import transformResponse from 'v2/ion/responseTransformer';
 import XHRFactorRequiredEmail from '../../../helpers/xhr/v2/FACTOR_REQUIRED_EMAIL';
-import XHRFactorVerificationRequiredEmail from '../../../helpers/xhr/v2/FACTOR_VERIFICATION_REQUIRED_EMAIL';
-import XHRTerminalTransfered from '../../../helpers/xhr/v2/TERMINAL_TRANSFERED';
 
 describe('v2/ion/responseTransformer', function () {
   it('returns result when invokes with invalid resp', () => {
     expect(transformResponse()).toBeNull();
     expect(transformResponse('hello')).toBeNull();
   });
 
-  it('converts factor require email', () => {
-    const result = transformResponse(XHRFactorRequiredEmail.response);
-    expect(result).toEqual({
-      'factor': {
-        'id': 'emf1axecbKovLJPWl0g4',
-        'factorType': 'email',
-        'provider': 'OKTA',
-        'vendorName': 'OKTA',
-        'profile': {
-          'email': 'e...a@rain.com'
-        }
-      },
-      'user': {
-        'id': 'I9bvFiq01cRFgbn',
-        'passwordChanged': '2019-05-03T19:00:00.000Z',
-        'profile': {
-          'login': 'foo@example.com',
-          'firstName': 'Foo',
-          'lastName': 'Bar',
-          'locale': 'en-us',
-          'timeZone': 'UTC'
-        }
-      },
-      'currentState': {
-        'version': '1.0.0',
-        'stateHandle': '01OCl7uyAUC4CUqHsObI9bvFiq01cRFgbnpJQ1bz82',
-        'expiresAt': '2018-09-17T23:08:56.000Z',
-        'step': 'FACTOR_REQUIRED',
-        'intent': 'login',
-        'submit-factor': jasmine.any(Function),
-        'cancel': jasmine.any(Function),
-        'context': jasmine.any(Function),
-        'remediation': [
+  it('converts factor required email idx object', () => {
+    const idxObjectFactorRequiredEmail  = {
```

Comment:
Wish we had jest to create snapshots. Can we move these to external helper files so it is easy to follow the test flow.